test/useFind.test.ts

```diff
@@ -935,4 +965,173 @@ describe('Find composition', () => {
+      mountComposition(() => {
+        findComposition = useFind('testModels');
+      });
+      await nextTick();
+
+      // then
+      expect(serviceFind).toHaveBeenCalledTimes(1);
+      expect(findComposition).toBeTruthy();
+      expect(findComposition && findComposition.data.value).toStrictEqual(testModels.slice(0, 1));
+    });
+
+    it('should load all data with chunking', async () => {
+      expect.assertions(3);
+
+      // given
+      let startItemIndex = 0;
+      const serviceFind = vi.fn(() => {
+        const page: Paginated<TestModel> = {
+          total: testModels.length,
+          skip: startItemIndex,
+          limit: 1,
+          data: testModels.slice(startItemIndex, startItemIndex + 1),
+        };
+        startItemIndex++;
+        return page;
+      });
+
+      const feathersMock = {
+        service: () => ({
+          find: serviceFind,
+          on: vi.fn(),
+          off: vi.fn(),
+        }),
+        on: vi.fn(),
+        off: vi.fn(),
+      } as unknown as Application;
+      const useFind = useFindOriginal(feathersMock);
+
+      // when
+      let findComposition = null as UseFind<TestModel> | null;
+      mountComposition(() => {
+        findComposition = useFind('testModels', undefined, { chunking: true });
+      });
+      await nextTick();
+
+      // then
+      expect(serviceFind).toHaveBeenCalledTimes(2);
+      expect(findComposition).toBeTruthy();
+      expect(findComposition && findComposition.data.value).toStrictEqual(testModels);
+    });
+
+    it('should also load chunked data with lastEvaluatedKey patterns (misused $skip for it)', async () => {
+      expect.assertions(3);
+
+      // given
+      const serviceFind = vi.fn((params?: Params) => {
+        const startItemIndex = testModels.findIndex(({ _id }) => _id === params?.query?.$skip) + 1;
+        const data = testModels.slice(startItemIndex, startItemIndex + 1);
+        const page: Paginated<TestModel> = {
+          total: testModels.length,
+          skip: data[data.length - 1]._id as unknown as number,
+          limit: 1,
+          data,
+        };
+        return page;
+      });
+
+      const feathersMock = {
+        service: () => ({
+          find: serviceFind,
+          on: vi.fn(),
+          off: vi.fn(),
+        }),
+        on: vi.fn(),
+        off: vi.fn(),
+      } as unknown as Application;
+      const useFind = useFindOriginal(feathersMock);
+
+      // when
+      let findComposition = null as UseFind<TestModel> | null;
+      mountComposition(() => {
+        findComposition = useFind('testModels', undefined, { chunking: true });
+      });
+      await nextTick();
+
+      // then
+      expect(serviceFind).toHaveBeenCalledTimes(2);
+      expect(findComposition).toBeTruthy();
+      expect(findComposition && findComposition.data.value).toStrictEqual(testModels);
+    });
+
+    it('should stop further page requests if find was retriggered due to e.g. reactivity', async () => {
```

Comment:
Should be more explicit IMO. What kind of reactivity do you have in mind? => change to `params`

Reply:
@anbraten Good point. I adjusted it